### DISCUSSION TITLE
⚠️ Binaries removed due to GPL license violation! ⚠️

### DISCUSSION THREAD

**Discussion by LegacyEffect** [Xaymar/obs-stream-effects#1097] [General]
What does this mean exactly.  Following the link directly from NVIDIA Broadcast Download.   Click on the link and see this big warning message.   Is it just removed from github?  Or only available via Patreon?  Never seen a link from a huge company like NVIDIA to get a big warning message like that.

**Comment by Xaymar** [comment]
The text means that the .exe/.pkg/.zip/.7z/... files had to be removed due to a legal issue with the contents. A past contributor had submitted code under their own name that they did not own the rights to. These binaries are also not available on Patreon either, they were completely removed.

There are currently no plans to recreate these binaries with the affected content removed.
